Gruntfile.coffee

```diff
@@ -109,6 +109,7 @@ module.exports = (grunt) ->
           'build/js/components/*.js'
           'build/js/directives/*.js'
           'build/js/filters/*.js'
+          'build/js/helpers/*.js'
```

Comment:
Just tested different scenarios with changing the dates. => Filename is updated accordingly, and the report is not generated if there is not transaction in the date range